### PR TITLE
honk

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

adds the clown radio headset with the new honk mode
image
_pssst... there may even be a mime headset_

## Why It's Good For The Game

**honk** 

## Changelog
:cl: Fikou
add: Due to riots on the streets, Nanotrasen gave clowns a special radio, it may even have a new mode!
add: Pssst, don't tell anybody, I heard mimes got a special radio as well...
/:cl:

### CODE REVIEW

**Comment by nemvar** [comment]
tfw no mime headset

**Comment by FantasticFwoosh** [comment]
Codersprites a bit later on then? I wanted a banana radio-phone.

**Comment by Fikou** [comment]
> tfw no mime headset

scared?

**Comment by Artemiuz28** [comment]
nice animu

**Comment by Ziiro** [comment]
clown powercreep, huge +1 from me

**Comment by bgobandit** [comment]
what's the point of the clown megaphone then

**Comment by Fikou** [comment]
> what's the point of the clown megaphone then

so you can speak loudly and proudly on anything that isnt the radio while the clown headset is only for radio

File: code/game/objects/items/devices/radio/headset.dm [hunk 1]
```diff
@@ -235,6 +237,18 @@ GLOBAL_LIST_INIT(channel_tokens, list(
...
+	name = "clown radio headset"
+	desc = "Headset used by the clown, the master of pranks and jokes."
+	icon_state = "clown_headset"
+	honk = TRUE
+
+/obj/item/radio/headset/headset_srv/mime
+	name = "mime radio headset"
+	desc = "Headset used by the mime, the master of silen- wait, why does he have one?"
+	icon_state = "mime_headset"
+	can_broadcast = FALSE
```
Comment (vuonojenmustaturska) [inline code/game/objects/items/devices/radio/headset.dm]: you could just override Hear for this headset

**Comment by Ratvar** [comment]
I like this because it's consistent compared to head headsets. You too can ask why captain needs loud mode when he has megaphone, bgobandit. Either go fully one way or another

**Comment by ExcessiveUseOfCobblestone** [comment]
remember when clowns were suppose to be funny?

**Comment by Ziiro** [comment]
no

**Comment by Fikou** [comment]
> remember when clowns were suppose to be funny?

does anyone

**Comment by optimumtact** [comment]
clown microphone already exists

**Pull request closed without merging**